### ISSUE TITLE
Postal Code Map popups

### ISSUE THREAD

**Issue by DanielHeidt** [TheCanadianConfederationDebates/TCCD#112]
When I click on a popup from the map and then click on the textual link to another riding (ex. Lower Fort Gary"), I am taken to http://hcmc.uvic.ca/confederation/en/canadaMap.html?place=LFOR1. Instead, the user should be linked to the LFOR1 node while remaining on http://hcmc.uvic.ca/confederation/en/index.html.

**Comment by martindholmes** [comment]
I disagree completely. It's impractical to run the full map functionality while we're also in the embedded scenario. Interacting with the map should shell out to the full-page map as soon as possible.
